drivers/mpu6050/MPU6050.hpp

```diff
@@ -56,7 +56,8 @@ class MPU6050: public ImuSensor
 	MPU6050(const char *device_path) :
 		ImuSensor(device_path, MPU6050_MEASURE_INTERVAL_US, false), // false = sensor has no mag
 		_last_temp_c(0.0f),
-		_temp_initialized(false)
+		_temp_initialized(false),
+		_packets_per_cycle_filtered(1.0)
```

Comment:
See the comment above. Yes, I do use the FIFO buffer, but at the moment the sampling and read frequencies match. Yet, there are a few cases where a packet is missed but read in the next cycle. In the future it would be possible to reduce the read frequency in the DriverFramework to 250Hz (the published frequency) and utilize the FIFO better.